### DISCUSSION TITLE
/dev/null

### DISCUSSION THREAD

**Discussion by pythonmandev** [xai-org/grok-1#154] [General]
/dev/null

**Comment by jihuishan** [comment]
Thanks for sharing! Now it's several MB per second instead of 100KB.

**Comment by chickun-io** [comment]
No, you can use BaiduNetDisk to download it because its download speed is really damn fast, up to 50KB per second! The size of this project is 300GB and it will only take you less than three months to complete the download operation! The best part is that you can download this 300G file for free without spending a dime!

**Reply by Rosangle** [reply]
哈哈哈

**Reply by m1n9o** [reply]
BaiduNetDisk is really fk awesome tool, we can even use the time it will take to download to train a model!
